src/Microsoft.Health.Fhir.Liquid.Converter/Filters/StringFilters.cs

```diff
@@ -122,5 +123,10 @@ public static string RegexReplace(string data, string pattern, string replacemen
         {
             return Regex.Replace(data, pattern, replacement);
         }
+
+        public static bool StartsWith(string data, string delimiter)
+        {
+            return data.StartsWith(data, ignoreCase: true, culture: CultureInfo.InvariantCulture);
```

Comment:
Is this relevant? https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings

> Do not use string operations based on StringComparison.InvariantCulture in most cases. One of the few exceptions is when you are persisting linguistically meaningful but culturally agnostic data.

Reply:
Ya I thought I was being smart here but it looks my brain needs to be updated, will fix.